examples/keep-alive-framework/src/main/scala/com/mesosphere/usi/helloworld/KeepAliveFramework.scala

```diff
@@ -0,0 +1,124 @@
+package com.mesosphere.usi.helloworld
+
+import java.util.UUID
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.stream.{ActorMaterializer}
+import akka.stream.scaladsl.{Flow, Sink}
+import com.mesosphere.mesos.client.MesosClient
+import com.mesosphere.mesos.conf.MesosClientSettings
+import com.mesosphere.usi.core.Scheduler
+import com.mesosphere.usi.core.models._
+import com.mesosphere.usi.core.models.resources.{ResourceType, ScalarRequirement}
+import com.typesafe.config.{Config, ConfigFactory}
+import com.typesafe.scalalogging.StrictLogging
+import org.apache.mesos.v1.Protos.{FrameworkInfo, TaskState, TaskStatus}
+
+import scala.concurrent.Await
+import scala.sys.SystemProperties
+import scala.concurrent.duration._
+
+class KeepAliveFramework(conf: Config) extends StrictLogging {
+
+  implicit val system = ActorSystem()
+  implicit val mat = ActorMaterializer()
+  implicit val ec = system.dispatcher
+
+  val settings = MesosClientSettings(conf.getString("master-url"))
+  val frameworkInfo = FrameworkInfo
+    .newBuilder()
+    .setUser(
+      new SystemProperties()
+        .get("user.name")
+        .getOrElse(throw new IllegalArgumentException("A local user is needed to launch Mesos tasks")))
+    .setName("KeepAliveFramework")
+    .addRoles("test")
+    .addCapabilities(FrameworkInfo.Capability.newBuilder().setType(FrameworkInfo.Capability.Type.MULTI_ROLE))
+    .setFailoverTimeout(0d)
+    .build()
+
+  val client: MesosClient = Await.result(MesosClient(settings, frameworkInfo).runWith(Sink.head), 10.seconds)
+
+  val runSpec = RunSpec(
+    resourceRequirements = List(ScalarRequirement(ResourceType.CPUS, 0.001), ScalarRequirement(ResourceType.MEM, 32)),
+    shellCommand = """echo "Hello, world" && sleep 5""",
+    role = "test"
+  )
+
+  def generatePodSpec() = {
+    val podId = PodId(s"hello-world.${UUID.randomUUID()}.1")
+
+    val podSpec = PodSpec(
+      id = podId,
+      goal = Goal.Running,
+      runSpec = runSpec
+    )
+
+    podSpec
+  }
+
+  val specsSnapshot = SpecsSnapshot(
+    podSpecs = (1 to 10).map(_ => generatePodSpec()),
```

Comment:
Please make an application parameter so that `numberOfPods` can be started by this example.

Reply:
➕